src/nvlink/worker_32bit.py

```
@@ -0,0 +1,377 @@
+
+                return {
+                    "success": ret_code == 0,
+                    "return_code": ret_code,
+                    "read_count": read_count,
+                    "download_count": dl_count,
+                    "timestamp": timestamp
+                }
+            else:
+                return {"success": False, "error": f"Unexpected result: {result}"}
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)